pkg/apis/options/logging.go

```diff
@@ -0,0 +1,74 @@
+package options
+
+import (
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/logger"
+	"github.com/spf13/pflag"
+)
+
+// LoggingOptions contains all options required for configuring the logging
+type LoggingOptions struct {
```

Comment:
Since this is in the options package, should this be named just `Logging`? Then it'll get refereed to as `options.Logging`. Currently it would stutter with `options.LoggingOptions`. But also this is consistent with `SessionOptions` and `CookieOptions`, so maybe it's better to leave it and be consistent for now :shrug:

Reply:
We could follow up to rename the others? I want to do PRs for separating their flags and validation too so could rename them then

Reply:
Yep, that sounds like a good plan.